docs/api/admin.yaml

```diff
@@ -112,20 +112,22 @@ paths:
     $ref: './paths/admin/{db}~_repair.yaml'
   /_all_dbs:
     $ref: ./paths/admin/_all_dbs.yaml
-  '/{db}/_compact':
-    $ref: './paths/admin/{db}~_compact.yaml'
+  '/{keyspace}/_compact':
+    $ref: './paths/admin/{keyspace}~_compact.yaml'
   '/{db}/':
     $ref: './paths/admin/{db}~.yaml'
+  '/{keyspace}/':
+    $ref: './paths/admin/{keyspace}~.yaml'
   /_expvar:
     $ref: ./paths/admin/_expvar.yaml
   /:
     $ref: ./paths/admin/~.yaml
   '/{db}/_all_docs':
     $ref: './paths/admin/{db}~_all_docs.yaml'
-  '/{db}/_bulk_docs':
-    $ref: './paths/admin/{db}~_bulk_docs.yaml'
-  '/{db}/_bulk_get':
-    $ref: './paths/admin/{db}~_bulk_get.yaml'
+  '/{keyspace}/_bulk_docs':
+    $ref: './paths/admin/{keyspace}~_bulk_docs.yaml'
+  '/{keyspace}/_bulk_get':
+    $ref: './paths/admin/{keyspace}~_bulk_get.yaml'
   '/{db}/_changes':
```

Comment:
It feels inconsistent from an API perspective to have _changes stay on db, but other multi-doc operations move to keyspace.  I understand that _changes will allow collection selection via parameters, but still seems a bit trickier to reason about as an API.